src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Decompression.cs

```diff
@@ -43,6 +51,15 @@ public static IEnumerable<object[]> DecompressedResponse_MethodSpecified_Decompr
         public async Task DecompressedResponse_MethodSpecified_DecompressedContentReturned(
             string encodingName, Func<Stream, Stream> compress, DecompressionMethods methods)
         {
+            if (!UseSocketsHttpHandler && encodingName == "br")
+            {
+                // Brotli only supported on SocketsHttpHandler.
+                return;
+            }
+
+            _output.WriteLine($"encodingName={encodingName}");
+            _output.WriteLine($"methods={methods}");
```

Comment:
Is this tracing really needed?  I'm not clear on what value this adds, as it's part of the theory name.

Reply:
I'll remove it.